Add support for installing i3wm desktop environment

JWorthe opened the pull request:
This installs the whole i3 group, so includes i3lock and i3status. I've also included a default terminal so that you have something that works out of the box, and lightdm so that you get a greeter.

I based this heavily on how I saw the other desktop environments like xfce were implemented.

tvararu commented on alis.conf:
```diff
@@ -72,7 +72,7 @@ HOOKS="base !udev !usr !resume !systemd !btrfs keyboard autodetect modconf block
 
 # desktop
-DESKTOP_ENVIRONMENT="!gnome !kde !xfce !mate !cinnamon !lxde" # (single)
+DESKTOP_ENVIRONMENT="!gnome !kde !xfce !mate !cinnamon !lxde !i3wm" # (single)
```
Would it make sense to add `i3-gaps` as well as an option?

JWorthe replied:
The [i3](https://www.archlinux.org/groups/x86_64/i3/) package group that I install includes both [i3-wm](https://www.archlinux.org/packages/community/x86_64/i3-wm/) and i3-gaps, and chooses [i3-gaps](https://www.archlinux.org/packages/community/x86_64/i3-gaps/) by default. So this would currently actually install i3-gaps.

It would be a simple enough change to offer both an i3-wm and an i3-gaps option if you think it's a good idea. I'm not sure if it would be adding more options just for the sake of more options? People who specifically want one or the other can also add i3-wm to one of the other package variables like PACKAGES_PACMAN_CUSTOM below.

JWorthe replied:
I see that @picodotdev had the same thought. I'll add it as two options

tvararu reviewed (approved):
I was going to investigate adding this tomorrow 👍

picodotdev commented on alis.sh:
```diff
@@ -1302,6 +1305,11 @@ function desktop_environment_lxde() {
 }
 
+function desktop_environment_i3wm() {
+    pacman_install "i3 dmenu rxvt-unicode lightdm lightdm-gtk-greeter xorg-server"
```
i3 package group contains i3-wm and i3-gaps and in the Arch Linux wiki there is the following message "Note: i3-wm conflicts with i3-gaps (a fork of i3 with gaps and other features) and by default i3-gaps will be installed."

Would be possible to allow the user to choose between i3-wm and i3-gaps when installing i3 package group? If it is necessary using in the DESKTOP_ENVIRONMENT these two different values i3wm and i3wm-gap to execute a different pacman_install command.

picodotdev reviewed (commented):
In the pull request this line should be changed to check that the user has configured in the configuration file a valid DESKTOP_ENVIRONMENT value.

https://github.com/picodotdev/alis/blob/88d974ffdd9bd09ec634e97efc5d02eb3eaf5f66/alis.sh#L163

JWorthe commented:
@picodotdev @tvararu I've added the choice between using the i3-wm and i3-gaps packages. I've also updated the docs wherever I found references to other desktop environments to also list i3-wm and i3-gaps. Please take another look.

picodotdev commented:
Great. Merged. A lot of thanks for the contribution!
